golden_response/staging/get_landing_page/county.json

```diff
@@ -409,36 +409,372 @@
     },
     "Count_Person_Employed": {
       "data": {
-        "1990": 818551,
-        "1991": 793935,
-        "1992": 789295,
-        "1993": 788965,
-        "1994": 798295,
-        "1995": 817464,
-        "1996": 859046,
-        "1997": 901592,
-        "1998": 921608,
-        "1999": 928555,
-        "2000": 912217,
-        "2001": 891097,
-        "2002": 813016,
-        "2003": 773878,
-        "2004": 768970,
-        "2005": 772046,
-        "2006": 784773,
-        "2007": 803877,
-        "2008": 816618,
-        "2009": 782384,
-        "2010": 830114,
-        "2011": 852426,
-        "2012": 882982,
-        "2013": 909040,
-        "2014": 938914,
-        "2015": 970215,
-        "2016": 989046,
-        "2017": 1005277,
-        "2018": 1015929,
-        "2019": 1027527
+        "1990-01": 817755,
```

Comment:
@beets I updated the ranking to be more deterministic based on all the observation properties. This gives denser and more recent data now.